### DISCUSSION TITLE
SO- Allow for wazuh error

### DISCUSSION THREAD

**Discussion by cerberus10748** [Security-Onion-Solutions/securityonion#8124] [Unsupported Versions] [answered]
Hello all, 
I am getting the following error when i try to run the so allow on a foward node for a wazuh agent i added and i get the following error:Traceback (most recent call last):
  File "/sbin/so-firewall", line 392, in <module>
    main()
  File "/sbin/so-firewall", line 382, in main
    code = cmd(options, args[1:])
  File "/sbin/so-firewall", line 303, in includehost
    result = addIp(args[0], args[1], 'insert')
  File "/sbin/so-firewall", line 101, in addIp
    content = loadYaml(hostgroupsFilename)
  File "/sbin/so-firewall", line 73, in loadYaml
    file = open(filename, "r")
FileNotFoundError: [Errno 2] No such file or directory: '/opt/so/saltstack/local/salt/firewall/hostgroups.local.yaml'
[c*****@node**** ~]$

please advise

**Comment by dougburks** [comment, marked as answer]
Please see https://docs.securityonion.net/en/2.3/firewall.html#managing.
